packages/block-library/src/navigation-menu/style.scss

```diff
@@ -106,4 +90,42 @@
 		}
 	}
 
+	// Menu event activation.
+	&.is-activated-by-hover li {
+		&:hover,
+		&:focus-within {
+			cursor: pointer;
+			z-index: 99999;
+		}
+
+		// Submenu Display
+		&:hover > ul,
+		&:focus-within > ul,
+		& ul:hover,
+		& ul:focus {
+			visibility: visible;
+			opacity: 1;
+			display: block;
+		}
+	}
+
+	&.is-activated-by-click li {
+		input.wp-block-navigation-menu-item__toggle-handler {
+			display: none;
+		}
+
+		label.wp-block-navigation-menu-item__toggle-for {
```

Comment:
I'd avoid qualifying with `label` unless it is required for specificity. Otherwise we're increasing the specificity for no reason which I would tend to avoid at all costs.

Ditto for `input.` above.